api/schemas/catalog.json

```diff
@@ -94,8 +94,88 @@
             "items": {
                 "type": "string"
             }
+        },
+        "metadata": {
+            "description": "A mapping of non-recursive File resources to their required metadata",
+            "type": "object",
+            "patternProperties": {
+                "^.+$": {
+                    "$ref": "#/definitions/file_metadata"
+                }
+            }
+        },
+        "recursive_metadata": {
+            "description": "A mapping of recursive File resources to a hash of sources and arrays of associated metadata for that resource",
+            "type": "object",
+            "patternProperties": {
+                "^.+$": {
+                    "type": "object",
+                    "patternProperties": {
+                        "^.+$": {
+                            "type": "array",
+                            "items": {
+                                "$ref": "#/definitions/file_metadata"
```

Comment:
no worries, sorry didn't mean that as a blocker for this PR. We can handle that as part of the docs ticket